index.html

```diff
@@ -31,24 +31,230 @@ <h1>Multi-Post<br> Stories</h1>
+              <a href="#">See Projects</a>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="story-wrapper">
+        <div class="work-section">
+          <div class="story-snapshoot-potofolio2"></div>
+          <div class="work-txt">
+            <h1 class="header-1">Multi-Post <br />Stories</h1>
+            <ul class="list-project-story">
+              <li class="first-element-ul">CANOPY</li>
+              <li><img src="/assets/images/Counter.png" alt="counter" /></li>
+              <li>Back End Dev</li>
+              <li><img src="/assets/images/Counter.png" alt="counter" /></li>
+              <li>2015</li>
+            </ul>
+            <div class="work-paragraph">
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+            </div>
+            <div class="work-buttons">
+              <ul>
+                <li class="html"><a href="#">html</a></li>
+                <li class="css"><a href="#">css</a></li>
+                <li class="js"><a href="#">javascript</a></li>
+              </ul>
+            </div>
+            <div class="see-project-btn">
+              <a href="#" class="get-resume-link">Get My Resume</a>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="about-wrapper">
+        <div class="text">
+          <h2>About <br />Myself</h2>
+          <p>
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+        </div>
+        <div class="social-media">
+          <div class="icon-a">
+            <a href="#">Let's connect</a>
+          </div>
+          <nav>
+            <a href="#">
+              <img
+                src="assets/images/twitter.svg"
+                alt="twitter image"
+                class="twitter"
+              />
+            </a>
+
+            <a href="#">
+              <img
+                src="assets/images/linkedin.svg"
+                alt="linked in  image"
+                class="linkedin"
+              />
+            </a>
+
+            <a href="#">
+              <img
+                src="assets/images/m.svg"
+                alt="meduim in  image"
+                class="meduim"
+              />
+            </a>
+
+            <a href="#">
+              <img
+                src="assets/images/github.svg"
+                alt="github in  image"
+                class="github"
+              />
+            </a>
+
+            <a href="#">
+              <img
+                src="assets/images/angleList.svg"
+                alt="angle list in  image"
+                class="anglelist"
+              />
+            </a>
+          </nav>
```

Comment:
- [ ] The `<nav>` tag defines a set of navigation links. In most cases, it is used on the navbar. It would be better if you remove it and wrap your icons in an unordered list: `<ul>` tag.

Reply:
use ul instead of nav bar